diff --git a/node_modules/yauzl-promise/lib/entry.js b/node_modules/yauzl-promise/lib/entry.js
index 3d55b73..20a0894 100644
--- a/node_modules/yauzl-promise/lib/entry.js
+++ b/node_modules/yauzl-promise/lib/entry.js
@@ -8,7 +8,7 @@
 // Modules
 const {createInflateRaw} = require('zlib'),
 	{Transform: TransformStream, pipeline} = require('stream'),
-	calculateCrc32 = require('@node-rs/crc32').crc32,
+       calculateCrc32 = (buf, seed) => require('crc-32').buf(buf, seed) >>> 0,
 	{isObject, isBoolean, isPositiveInteger, isPositiveIntegerOrZero} = require('is-it-type'),
 	assert = require('simple-invariant');
 
diff --git a/node_modules/yauzl-promise/lib/zip.js b/node_modules/yauzl-promise/lib/zip.js
index 456259f..6c669cb 100644
--- a/node_modules/yauzl-promise/lib/zip.js
+++ b/node_modules/yauzl-promise/lib/zip.js
@@ -8,7 +8,7 @@
 'use strict';
 
 // Modules
-const calculateCrc32 = require('@node-rs/crc32').crc32,
+const calculateCrc32 = (buf, seed) => require('crc-32').buf(buf, seed) >>> 0,
 	assert = require('simple-invariant'),
 	{isPositiveIntegerOrZero} = require('is-it-type');
 
diff --git a/node_modules/yauzl-promise/package.json b/node_modules/yauzl-promise/package.json
index 40721c5..e9c6386 100644
--- a/node_modules/yauzl-promise/package.json
+++ b/node_modules/yauzl-promise/package.json
@@ -17,7 +17,6 @@
     "url": "https://github.com/overlookmotel/yauzl-promise/issues"
   },
   "dependencies": {
-    "@node-rs/crc32": "^1.7.0",
     "is-it-type": "^5.1.2",
     "simple-invariant": "^2.0.1"
   },
@@ -32,6 +31,9 @@
     "jest-runner-eslint": "^2.0.0",
     "npm-run-all": "^4.1.5"
   },
+  "peerDependencies": {
+    "crc-32": "^1.2.2"
+  },
   "keywords": [
     "yauzl",
     "unzip",
